daml-assistant/integration-tests/BUILD.bazel

```
@@ -27,17 +27,16 @@ genrule(
       set -euo pipefail
       TMP_DIR=$$(mktemp -d)
       MVN_DB="$$TMP_DIR/m2"
-      VERSION=$$(cat $(location //:component-version))
-      install_mvn() {
+      install_mvn() {{
           $(location @mvn_dev_env//:mvn) -q install:install-file \
           -Dmaven.repo.local=$$MVN_DB \
           "-DgroupId=$$1" \
           "-DartifactId=$$2" \
           "-Dfile=$$3" \
           "-DpomFile=$$4" \
           -Dpackaging=jar \
-          -Dversion=$$VERSION
-      }
+          -Dversion={}
```

Unnamed placeholders scare me. (Maybe I haven't used enough C-style `printf` in my life?)
```suggestion
          -Dversion={mvn_version}
```
There are a few more instances of this elsewhere.